spec/models/case_attachment_upload_group_spec.rb

```diff
@@ -1,11 +1,9 @@
 require "rails_helper"
 
-# rubocop:disable RSpec/InstanceVariable, RSpec/BeforeAfterAll
+# rubocop:disable RSpec/InstanceVariable
 describe CaseAttachmentUploadGroup do
-  before(:all) do
-    DbHousekeeping.clean
-
-    @upload_group = "20170608101112"
+  before do
+    @upload_group = "20170608111112"
```

Comment:
You should define these variables using let instead outside of the before block.
You can then also remove the rubocop rule disabling on lines 3 and 80.